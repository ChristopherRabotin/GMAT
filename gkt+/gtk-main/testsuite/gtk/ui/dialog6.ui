<!-- test invalid id in dialog markup -->
<interface>
  <object class="GtkDialog">
    <action-widgets>
      <action-widget response="ok">xyzzy</action-widget>
    </action-widgets>
  </object>
</interface>
