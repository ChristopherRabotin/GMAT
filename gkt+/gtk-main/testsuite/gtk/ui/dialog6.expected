ERROR: gtk-builder-error-quark 13
.:5:37 Object with ID xyzzy not found
